src/SFA.DAS.EmployerIncentives.Commands/Services/LearnerMatchApi/LearnerService.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Newtonsoft.Json;
+using SFA.DAS.EmployerIncentives.Domain.ApprenticeshipIncentives.ValueTypes;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.EmployerIncentives.Commands.Services.LearnerMatchApi
+{
+    public class LearnerService : ILearnerService
+    {
+        private readonly HttpClient _client;
+        private readonly string _serviceVersion;
+
+        public LearnerService(
+            HttpClient client,
+            string serviceVersion)
+        {
+            _client = client;
+            _serviceVersion = serviceVersion;
+        }
+
+        public async Task Refresh(Learner learner)
+        {
+            var response = await _client.GetAsync($"api/v{_serviceVersion}/{learner.Ukprn}/{learner.UniqueLearnerNumber}?");
+
+            if (response.StatusCode == System.Net.HttpStatusCode.NotFound)
+            {
+                learner.SetSubmissionData(null);
+
+                return;
+            }
+
+            response.EnsureSuccessStatusCode();
+
+            var jsonString = await response.Content.ReadAsStringAsync();
+            var learnerSubmissionDto = JsonConvert.DeserializeObject<LearnerSubmissionDto>(jsonString);
```

Comment:
We need something somewhere (maybe not here) to check if the ILR submission date is the same as the record we already have stored.  No point performing any of the processing if it hasn't changed.

Reply:
Could this be a new story, as I think it will require us to store additional values prior to the update.

Reply:
follow-up story?